curved_bottom_navigation/src/main/java/np/com/susanthapa/curved_bottom_navigation/CurvedBottomNavigationView.kt

```diff
@@ -406,30 +406,32 @@ class CurvedBottomNavigationView @JvmOverloads constructor(
     }
 
     fun onMenuItemClick(index: Int) {
+        var shouldAnimate = true;
```

Comment:
Just a thought, wouldn't it be simpler if we just move the `menuItemClickListener?.invoke(cbnMenuItems[index], index)` here as it would get triggered anyways.

Reply:
I thought of that, but since you put the invoke line at the end of it all I imagined some things had to happen before it (and did not delve too much to check if that was the case). If that's not the case then surely it's way easier/simpler to simple put it as the first line!

Reply:
Should I do another PR that way, then?

Reply:
Okay, PR simplified! (sorry for the mess, this is my first PR ever lol)